lib/ext/kafka.rb

```diff
@@ -130,6 +130,21 @@ def self.get_broker_metadatas(zk_client, brokers, force_rack = true)
       )
       Scala::Collection::JavaConversions.seq_as_java_list(broker_metadatas).to_a
     end
+
+    def self.get_broker_rack(zk_client, broker_id)
+      broker_metadata = Kafka::Admin.get_broker_metadatas(zk_client, [broker_id]).first
+      rack = broker_metadata.rack
+      unless rack.isDefined
+        raise "Broker #{broker_metadata.id} is missing rack information, unable to create rack aware shuffle plan."
+      end
+      rack.get
+    rescue Java::KafkaAdmin::AdminOperationException => e
+      if e.message.match '--disable-rack-aware'
```

Comment:
`#match` is for regexes, and although it works it feels like `#include?` is more appropriate here.